src/birdie/internal/titles.gleam

```diff
@@ -269,15 +272,18 @@ fn snap_call(
         left: title,
         right: glance.Call(
           function:,
-          arguments: [glance.Field(Some("content"), _snapshot_content)],
+          arguments: [glance.LabelledField("content", _snapshot_content)],
         ),
       )
     | // A call to the `birdie.snap` function where the content is piped into
       // it and the title is passed as an argument - labelled or not.
       glance.BinaryOperator(
         name: glance.Pipe,
         left: _snapshot_content,
-        right: glance.Call(function:, arguments: [glance.Field(_, title)]),
+        right: glance.Call(
+          function:,
+          arguments: [glance.LabelledField(_, title)],
```

Comment:
This changes the matching, now it matches only on labelled fields, while previously it would match on both labelled and unlabelled ones

Reply:
Added another case here aswell